PC: Implementación de pagos de vida-#109

dierodcam opened the pull request:
Se ha implementado el poder realizar pagos de vidas junto a las suscripciones mensuales.

dierodcam committed:
Pago de vidas-#109

Se han añadido los modelos de pago de vidas y se ha avanzado en los controladores

rafsalcas1 committed:
Implementación compras de vidas-#109

Hemos implementado la posibilidad de comprar distintos tipos de paquetes de vidas mediante stripe.

rafsalcas1 committed:
Adicción de las migraciones-#109

Hemos añadido las migraciones de tienda ya que no se han subido anteriormente y ha dado fallos en travis.

rafsalcas1 committed:
Integración de pago vidas y suscripción-#109

Hemos integrado los pagos de las vidas junto a las de suscripción de manera que ambos pertenezcan al mismo modelo.

ferruirob commented:
He detectado un error al hacer un pago con productos con un importe inferior a 0.5 euros. Solucionadlo por favor. Gracias

rafsalcas1 committed:
Solución del modelo producto-#109

Se ha solucionado un error relacionado con el valor mínimo del producto, ya que stripe no permite pagos inferiores a 50 cents.

dierodcam commented:
Problema solucionado @ferruirob

ferruirob reviewed (approved):
Probado los cambios, apruebo la solicitud de merge. Gracias

javsolgar commented:
Se ha detectado un error en el perfil de usuario a la hora de realizar un pago, se sube a continuación la corrección de este problema. Gracias.

javsolgar committed:
Se ha corregido un error al realizar el pago-#109

Se ha corregido un error a la hora de pagar por una suscripción, además, se ha añadido una redirección al perfil cuando el pago a finalizado con éxito

ferruirob committed:
Cambio para suscripción PC-#109

javsolgar committed:
Se ha completado el pago en Stripe con recurrencias-#109

Se ha completado el pago de suscripciones, y sus cancelaciones en Stripe, además de mejoras en las view

javsolgar committed:
Añadido json con los productos-#109

Se ha añadido un json con los productos iniciales, para que puedan cargarse a la base de datos comodamente

fragonlug reviewed (approved):
Se ha comprobado satisfactoriamente la posibilidad de suscribirte, de cancelar la suscrición y del pago y uso de las vidas compradas.

jorandmol reviewed (approved):
He podido comprobar que todo funciona como se espera, por lo que apruebo esta petición.
